files/default/neutron-ha-tool.py

```diff
@@ -81,6 +81,9 @@ def parse_args():
                     help='Replicate DHCP configuration to all agents')
     ap.add_argument('--now', action='store_true', default=False,
                     help='Migrate Routers immediately without a delay.')
+    ap.add_argument('--wait-for-ports', action='store_true', default=False,
```

Comment:
It's more important that the script has the desired effect than that it finishes ASAP.  And we already know that in production clouds, allowing a backlog to build in the OVS agents and/or neutron servers can cause the whole thing to fall over in a spectacular heap.  Correctness should trump performance.